package/firmware/ipq-wifi/Makefile

```diff
@@ -26,6 +26,7 @@ endef
 ALLWIFIBOARDS:= \
 	avm_fritzrepeater-1200 \
 	linksys_ea8300 \
+	mobipromo_cm520-79f \
```

Comment:
Please follow the instructions above in the comment and post the vendors boardfiles to the ath10k mailing list

>
> This is intended to be used on an interim basis until device-specific
> board data for new devices is available through the upstream compilation
>
> Please send a mail with your device-specific board files upstream.
> You can find instructions and examples on the linux-wireless wiki:
> <https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles>
>

Reply:
Board files are now sent to the ath10k mailing list.